cmd/install/install.go

```diff
@@ -0,0 +1,229 @@
+package install
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	"runtime"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+var (
+	ErrKubectlInstall  = errors.New("kubectl is not installed")
+	ErrKubeconfigNs    = errors.New("set the kubeconfig or kubeconfig does not have required permissions")
+	ErrArgoCD          = errors.New("argocd is not installed")
+	ErrArgoCDAuthToken = errors.New("argocd auth token has expired, login to argocd again")
+	ErrGit             = errors.New("git is not installed")
+	ErrKubectlPresent  = errors.New("kubectl is already installed")
+	ErrGitPresent      = errors.New("git is already installed")
+	ErrArgoCDPresent   = errors.New("argocd is already present")
+	ErrKubectlFail     = errors.New("error installing kubectl")
+	ErrArgoCDFail      = errors.New("error installing argocd")
+	kubectlPath        = "/usr/local/bin/kubectl"
+	argocdPath         = "/usr/local/bin/argocd"
+	Yellow             = color.New(color.FgHiYellow).SprintFunc()
+	Green              = color.New(color.FgGreen).SprintFunc()
+	Red                = color.New(color.FgRed).SprintFunc()
+)
+
+func NewCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:               "install",
+		Short:             "Install required tools for Arlon",
+		Long:              "Install kubectl, Argocd cli, check git cli",
+		DisableAutoGenTag: true,
+		Example:           "arlonctl install",
+		RunE: func(c *cobra.Command, args []string) error {
+			var err error
+			// Install kubectl and point it to the kubeconfig
+			_, err = installKubectl()
+			if err == ErrKubectlPresent {
+				fmt.Println(Green("✓") + "kubectl is already present in the path")
+			} else if err != nil {
+				fmt.Println(Red("x ")+"Error while installing kubectl ", err)
+			} else {
+				fmt.Println(Green("✓") + "Successfully installed kubectl")
+			}
```

Comment:
I was thinking the user should know what requirements are missing and so I was going forward without exiting once there is an error in this command.
The user can always run `verify` and thus come to know what requirements are missing and so then this can error out if install fails

Reply:
Yeah, that seems to be totally fine as well, but do try and see if it's possible to simplify the conditional statements.

For now, marking this as resolved.